pkg/crd/generator/testData/config/crds/fun_v1alpha1_toy.yaml

```diff
@@ -28,7 +28,9 @@ spec:
   group: fun.myk8s.io
   names:
     kind: Toy
-    plural: toys
+    plural: services
+    shortNames:
+    - ty
```

Comment:
Do we have test case where we don't have any resource annotations ?

Reply:
Yes but I'm not sure how to include another test case(without resource annotation) in the same file.

Reply:
https://gist.github.com/sohankunkerkar/0e814af32d07da11827d39151a342429 (toy_types.go)
https://gist.github.com/sohankunkerkar/f1007b71d101a571ea7ccb356955c521 (crd)